front/src/modules/rollingStock/helpers/utils.ts

```diff
@@ -507,3 +507,54 @@ export const handleUnitValue = <U extends MultiUnit>(
   }
   return +valueToConvert;
 };
+
+/**
+ * Updates a value expressed in a mass dependent unit when the mass change
+ * so that the value expressed in a mass independent value does not change.
+ * We assume mass dependent units only depend on the mass as 'MultiUnit/t'.
+ */
+export const rescaleMassDependentValue = (
```

Comment:
nit : we don't import this function anywhere
```suggestion
const rescaleMassDependentValue = (
```

Reply:
Thanks, done! ^^